x-pack/plugins/security_solution/cypress/test_plans/detection_response/prebuilt_rules/prebuilt_rules_install_update_workflows.mdx

```diff
@@ -0,0 +1,184 @@
+
+**GIVEN** user has the latest version of prebuilt rules `<prebuilt_rules_status>`  
+**WHEN** user navigates to the Rules Management Page  
+**THEN** no callout message is displayed for `<prebuilt_rules_status>` rules  
+
+*CASE 1: `<prebuilt_rules_status>` = installed*  
+*CASE 2: `<prebuilt_rules_status>` = updated*
+
+#### **Scenario: Callout message is displayed when there are new prebuilt rules available to install**
+
+**GIVEN** user already has 8.x prebuilt rules installed  
+**AND** there are new prebuilt rules available to install  
+**WHEN** user navigates to the Rules Management Page  
+	**THEN** user should see a callout message to install new prebuilt rules  
+**AND** the number of new rules available to install should be displayed on the +Add Elastic Rules link  
+
+#### **Scenario: Callout message is displayed when there are new updates on already installed prebuilt rules**
+
+**GIVEN** user already has 8.x prebuilt rules installed  
+**AND** there are new updates available for those prebuilt rules  
+**WHEN** users navigate to the Rules Management Page  
+**THEN** users should see an update callout message  
+**AND** the number of outdated rules should be displayed on the Rules Updates tab  
+
+#### **Scenario: User is notified of available prebuilt rules to install when a rule is deleted**
+
+**GIVEN** user has the latest version of prebuilt rules installed  
+**WHEN** user navigates to Rules Management Page  
+**AND** user deletes some prebuilt rules  
+**THEN** user should see a callout message with the same amount of prebuilt rules ready to install  
+
+
+
+
+### Prebuilt Rules Installation
+
+#### **Scenario: User without any installed prebuilt rule can install `<amount>` prebuilt rules**
+
+**GIVEN** a user that doesn’t have prebuilt rules installed  
+**WHEN** user navigates to Add Elastic Rules Page  
+**THEN** available prebuilt rules are displayed on Elastic Rules table  
+**AND** user can install `<amount>` prebuilt Rules  
+**AND** successfully installed message is displayed after installation  
+**AND** installed rules are removed from Elastic Rules table  
+**AND** rules to install counter is decreased accordingly  
+
+*CASE 1: `<amount>` = All*  
+*CASE 2: `<amount>` = Selected*  
+
+#### **Scenario: User performing a clean install for prebuilt rules sees a loading skeleton until installation is completed**
+
+**GIVEN** a user that is on Rules Management Page  
+**WHEN** user installs all prebuilt rules through Add Elastic Rules button/link  
+**THEN** a loading skeleton is displayed until the installation is completed
+
+
+
+
+
+### Prebuilt Rules Update
+
+#### **Scenario: Users can update prebuilt rules**
+
+**GIVEN** user already has 8.x prebuilt rules installed in Kibana  
+**AND** there are new updates available for those prebuilt rules  
+**AND** user is on Rules Management Page  
+**WHEN** user navigates to the Rules Update tab  
+**THEN** user should see all the prebuilt rules  
```

Comment:
```**THEN** user should see all the prebuilt rules```

This line is not right, only rules with updates should be displayed. Let's remove it.

Reply:
Synced with @vgomez-el:

We'll replace the line with:
```
**THEN** user should see all the prebuilt rules that have updates available
```